src/include/mpitimpl.h

```diff
@@ -1238,5 +1241,48 @@ extern MPID_Thread_mutex_t mpi_t_mutex;
 /* Init and finalize routines */
 extern int MPIR_T_env_init(void);
 extern void MPIR_T_env_finalize(void);
+extern void MPIR_T_events_finalize(void);
+
+typedef MPI_Count(MPIR_T_timestamp_fn) (void);
```

Comment:
Is this the same as 
`typedef MPI_Count (*MPIR_T_timestamp_fn)(void);`
?`

Reply:
Yes, can modify and make it clearer.